include/datadog_agent_six.h

```diff
@@ -33,6 +33,8 @@ DATADOG_AGENT_SIX_API void release_gil(six_t *, six_gilstate_t);
 DATADOG_AGENT_SIX_API int get_check(six_t *, const char *name, const char *init_config, const char *instances,
                                     six_pyobject_t **check, char **version);
 DATADOG_AGENT_SIX_API const char *run_check(six_t *, six_pyobject_t *check);
+DATADOG_AGENT_SIX_API void six_free(six_t *, void *ptr);
```

Comment:
why not directly call `C.free()` ?

Reply:
because, at least on windows, each module has its own heap.  You can't free memory in one module that's allocated from another (unless you use GetProcessHeap()).

This was added after a discussion I had with @masci about various leaks & potential crashes I identified, at least on windows.

Reply:
Good to know, thanks !